modules/gamedev/blog.py

```diff
@@ -2,7 +2,56 @@
 
 
 class Blog(glados.Module):
+    def setup_memory(self):
+        self.memory['data path'] = os.path.join(self.data_dir, 'blog')
+        if not os.path.isdir(self.memory['data path']):
+            os.mkdir(self.memory['data path'])
+        self.memory['data file'] = os.path.join(self.memory['data path'], 'submissions.txt')
+        self.memory['count file'] = os.path.join(self.memory['data path'], 'count.txt')
+
     @glados.Module.command('blog', '', 'Returns a link to the chat blog')
     async def blog(self, message, channel):
         await self.client.send_message(message.channel, 'http://gdnetchat.tumblr.com/submit')
 
+    @glados.Module.command('blogpost', '', 'Submits the current conversation to the chat blog')
+    async def blogpost(self, message, channel):
+        with codecs.open(self.memory['count file'], 'r', encoding='utf-8') as f:
```

Comment:
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/thecomet/documents/programming/python/GLaDOS2/env/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/thecomet/documents/programming/python/GLaDOS2/glados/bot.py", line 105, in on_message
    await callback(message, content)
  File "modules/gamedev/blog.py", line 20, in blogpost
    with codecs.open(self.memory['count file'], 'r', encoding='utf-8') as f:
  File "/home/thecomet/documents/programming/python/GLaDOS2/env/lib64/python3.6/codecs.py", line 895, in open
    file = builtins.open(filename, mode, buffering)
FileNotFoundError: [Errno 2] No such file or directory: 'data/334046991783034883/blog/count.txt'